crates/tokmd-core/CLAUDE.md

```
@@ -44,14 +46,20 @@ let lang = LangArgs {
 let receipt = scan_workflow(&global, &lang, Some(RedactMode::Paths))?;
 ```
 
-## Redaction Support
+### Redaction Support
 
 | Mode | Behavior |
 |------|----------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)